docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+:::
+
+### Data source description
+
+Describes the source of the data or parameter (e.g., *Calibration*)
+
+### Simulation software description
+
+Describes the simulation software (e.g., *sim_telarray* or *corsika*) the parameter is used for.
+
+```yaml
+simulation_software:
+  - name: corsika
+```
+
+Allowed is here to add the internal simulation software parameter naming (if different than the name used here in the schema).
+This is important to write correct and consistent CORSIKA or sim\_telarray configuration files.
+
+ Example:
+
+```yaml
+simulation_software:
+  - name: sim_telarray
+    internal_parameter_name: secondary_ref_radius
+```
+
+## Data structure for model parameters
+
+The model parameters are stored in json-style in the model repository and database.
+A typical model parameter file looks like:
+
+```json
+{
+    "parameter": "num_gains",
+    "instrument": "LSTN-01",
+    "site": "North",
+    "version": "2024-02-01",
+    "value": 2,
+    "unit": null,
+    "type": "int64",
+    "applicable": true,
+    "file": false
+}
+```
+
+## Updating the model database
+
+:::{Danger}
+This is for experts only and might impact the simulation model database for all users.
+Tests should be done before applying this to the production databases.
+:::
+
+### Update a single model parameter
+
+New model parameter defined in the simtools database format (json file) can be uploaded to the database using the {ref}`simtools-add-value-from-json-to-db <add_value_from_json_to_db>` application.
+New data files can be uploaded using {ref}`simtools-add-file-to-db <add_file_to_db>`.
+
+### Upload a complete simulation model from model repository to model database
+
+The application `add_model_parameters_from_repository_to_db.py` allows to export the simulation model parameters from a
+model repository to the model database for a given model versions.
+See the [database](databases.md#databases) section for implementation details on databases in simtools.
+
+Example:
+
+```bash
+simtools-add_model-parameters-from-repository-to-db \
+      --input_path /path/to/repository \
+      --db_name new_db_name
+```
+
+This application loops over all subdirectories in `input_path` and uploads all json files to the database `new_db_name` (or updates an existing database with the same name):
+
+- subdirectories starting with `OBS` are uploaded to the `sites` collection
+- json files from the subdirectory `configuration_sim_telarray` are uploaded to the `configuration_sim_telarray` collection
+- `Files` are added to the `files` collection
+- all other json files are uploaded to collection defined in the array element description in [simtools/schemas/array_elements.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/array_elements.yml). Allowed values are e.g., `telescopes`, `calibration_devices`.
+
+## Import simulation model parameters
+
+### Import Prod6 model parameters
```

Comment:
Also in the titles I would add "from `sim_telarray`"

Reply:
It is from sim_telarray and corsika (and I don't like long title :-))